scripts/dockutil

```diff
@@ -543,10 +543,6 @@ def addItem(pl, add_path, replace_label=None, position=None, before_item=None, a
     if showas == None: showas = 0
     if arrangement == None: arrangement = 2
 
-    #fix problems with unicode file names
-    enc = (sys.stdin.encoding if sys.stdin.encoding else 'UTF-8')
```

Comment:
I will need to test with emojis etc.

Reply:
Python 3 handles Unicode much better than 2.7, but definitely did not test items with emojis